src/SetCriteriaHeader.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+class SetCriteriaHeader extends React.Component {
+    render () {
+	return (
+		<div>
+		<h1>Set Judging Round Criteria </h1>
```

Comment:
this doesn't functionally do anything other than render static html. is it necessary to break out into a component?